src/sage/rings/finite_rings/finite_field_base.pyx

```diff
@@ -2120,6 +2120,91 @@ cdef class FiniteField(Field):
         python_int = int.from_bytes(input_bytes, byteorder=byteorder)
         return self.from_integer(python_int)
 
+    def _roots_univariate_polynomial(self, f, ring, multiplicities, algorithm=None):
+        r"""
+        Return the roots of the univariate polynomial ``f``.
+
+        INPUT:
+
+        - ``f`` - a polynomial defined over this field
+
+        - ``ring`` - the ring to find roots in.
```

Comment:
```suggestion
        - ``f`` -- a polynomial defined over this field

        - ``ring`` -- the ring to find roots in
```

and similar below (no full stop if there's only one noun phrase instead of full sentences)